src/components/navigation-tray/nav-tray.jsx

```diff
@@ -0,0 +1,71 @@
+/// This component is a navigation tray
+/// for mobile version of the app
+
+import { NavLink } from "react-router-dom";
+import notificationSign from "../../assets/notification_sign.svg";
+import styled from "styled-components";
+
+/// Here is a styles for every link:
+
+const StyledNavLink = styled(NavLink)`
+  width: 60px;
+  height: 60px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: center;
+  position: relative;
+  background-image: url("${({ inactive }) => inactive}");
+  &:active {
+    background-color: rgba(255, 255, 255, 0.3);
+  }
+  &.active {
+    background-color: rgba(255, 255, 255, 0.3);
+    background-image: url("${({ active }) => active}");
+  }
+`;
+
+const StyledNavBar = styled.nav`
+  width: 215px;
+  height: 60px;
+  background-color: white;
+  bottom: 3%;
+  right: 50%;
+  transform: translate(50%, 0);
+`;
+
+let num = 0;
+
+export const NavTray = ({ isUnreadedMessages, navLinks }) => {
+  return (
+    <StyledNavBar
+      className="nav position-fixed d-flex justify-content-evenly align-items-center rounded-pill bg-white"
+      style={{ "--bs-bg-opacity": 0.1 }}
+    >
+      {navLinks.map((link) => {
+        return (
+          <StyledNavLink
+            key={(num += 1)}
+            role={link.role}
+            to={link.to}
+            className="nav-link rounded-circle"
+            inactive={link.icon}
+            active={link.activeIcon}
+          >
+            {link.isNotification ? (
```

Comment:
nit: I like `hasNotification` as opposed to `isNotification`. Thoughts?

Reply:
Additionally, just a tip-- instead of doing `condition ? (thing) : ""`, you can simply do `condtion && (thing)`, that way you don't need the empty quotes in the case that you don't want to render anything for the `else` branch of the ternary.

Reply:
The reason I used isNotification instead of hasNotification is this property exists for "is navigation link a notification or not?". Because some of the navigation links might not require to have notification alerts on them (ex. home link does not need an alert). Moreover for hasNotification there is property on NavTray component, isUnreadMessages, which can't be triggered by setState hook somewhere in the code if the user receive a notification. Let me know what you think.